browser/resources/settings/brave_ipfs_page/brave_ipfs_page.html

```diff
@@ -64,18 +64,11 @@
         label="$i18n{ipfsAutoFallbackToGatewayLabel}"
         sub-label="$i18n{ipfsAutoFallbackToGatewayDesc}">
     </settings-toggle-button>
-    <settings-toggle-button id="ipfsAutoRedirectGateway"
+    <settings-toggle-button id="ipfsAutoRedirectToConfiguredGateway"
       class="cr-row"
-      pref="{{prefs.brave.ipfs.auto_redirect_gateway}}"
-      label="$i18n{ipfsAutoRedirectGatewayLabel}"
-      sub-label="$i18n{ipfsAutoRedirectGatewayDesc}">
-    </settings-toggle-button>
-    <settings-toggle-button id="ipfsAutoRedirectDNSLink"
-      class="cr-row"
-      pref="{{prefs.brave.ipfs.auto_redirect_dnslink}}"
-      label="$i18n{ipfsAutoRedirectDNSLinkLabel}"
-      sub-label="$i18n{ipfsAutoRedirectDNSLinkDesc}"
-      learn-more-url="$i18n{ipfsDNSLinkLearnMoreURL}">
```

Comment:
should we keep the "learn more" url?

Reply:
And if not, do we need to remove something from the help pages?